src/main/java/org/spongepowered/common/config/type/GlobalConfig.java

```diff
@@ -58,46 +58,51 @@
     @Setting(comment = "Configuration options related to the Sql service, including connection aliases etc")
     private SqlCategory sql = new SqlCategory();
 
-    @Setting
+    @Setting(comment = "Configuration options related to commands, including command aliases and hidden commands.")
     private CommandsCategory commands = new CommandsCategory();
 
-    @Setting
+    @Setting(comment = "Configuration options related to permissions and permission handling")
     private PermissionCategory permission = new PermissionCategory();
 
-    @Setting(value = "modules")
+    @Setting(value = "modules", comment = "Options related to optional modules, that can be enabled or disabled.")
```

Comment:
I think it's worth explaining what modules are a bit more. For example:

```
Sponge provides a number of modules that allow for enabling or disabling\n
certain specific features. These may be enabled or disabled below.\n\n
Any changes here require a server restart as modules are applied at startup.
```

Reply:
Changed